crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -55,7 +55,12 @@ use std::{cell::UnsafeCell, marker::PhantomData};
 /// - Methods can be implemented for the query items.
 /// - There is no hardcoded limit on the number of elements.
 ///
-/// This trait can only be derived if each field of the struct also implements `WorldQuery`.
+/// This trait can only be derived if each field either
+///
+/// * also implements `WorldQuery`, or
+/// * is marked with `#[world_query(ignore)]`. Fields decorated with this attribute
+///   will be created with the default value upon realisation.
```

Comment:
Please explicitly mention that `Default` is both required to be implemented and used in this case.

Reply:
Not 100% sure what you mean by `Default` is [...] required to be [...] used`. Can you elaborate? For now I've only added that it needs to be implemented for the type.